tensorflow/contrib/lite/examples/ios/simple/RunModelViewController.mm

```diff
@@ -118,52 +113,51 @@ - (UIImage *)fixrotation:(UIImage *)image{
     
     // Now we draw the underlying CGImage into a new context, applying the transform
     // calculated above.
-    CGContextRef ctx = CGBitmapContextCreate(NULL, (size_t) image.size.width, (size_t) image.size.height,
+    CGContextRef ctx = CGBitmapContextCreate(NULL, (size_t)image.size.width, (size_t) image.size.height,
```

Comment:
I don't think you need size_t casts, and there is a space between (size_t) and image.size.height